### ISSUE TITLE
[CI] m2gl025_miv in Shippable CI systematically fails some tests

### ISSUE THREAD

**Issue by pfalcon** [zephyrproject-rtos/zephyr#25017]
Apparently, recently, a real hardware board `m2gl025_miv` was added to Shippable CI (vs running just in QEMU). I'm not sure how recently, as I never saw a public announcement for such an important event as adding a real hardware to project CI testing.

The matter, it looks like the complete set of tests wasn't run on this board, and now touching some tests (which causes them to be run in CI), leads to systematic failures. This is examplified by https://github.com/zephyrproject-rtos/zephyr/pull/25000, where making a dummy change to tests/posix/common causes it to fail. (This was discovered while working on https://github.com/zephyrproject-rtos/zephyr/pull/24964).

**Comment by pfalcon** [comment]
I'm not sure to whom assign this issue, so assigning to @nashif as the overall owner of the project infra and @carlescufi, on whose request https://github.com/zephyrproject-rtos/zephyr/pull/24964 was prepared, which is now blocked by this issue.

**Comment by pfalcon** [comment]
I'd say systematically failing CI is "priority: high", but as there're no proofs that too many posted PRs are affected by this, setting as medium.

**Comment by pfalcon** [comment]
Fix merged into https://github.com/zephyrproject-rtos/zephyr/pull/24964. Closing.
